packages/web-components/fast-foundation/src/anchored-region/anchored-region.ts

```diff
@@ -18,6 +18,7 @@ declare global {
 }
 
 export type AxisPositioningMode = "uncontrolled" | "locktodefault" | "dynamic";
+export type AxisScalingMode = "anchor" | "availablespace" | "content";
```

Comment:
Thoughts on changing "availablespace" to be "grow"?

Reply:
Hmmm....not sure if that's any clearer.  But I think "availablespace" could be improved on.

Reply:
My concern wasn't so much clarity as verbosity. Both seem equally clear to me but "grow" is much less verbose. Totally open to other ideas though.

Reply:
"scale"?

Reply:
"fill" could work also maybe

Reply:
"fill" is the best yet I think

Reply:
Going with "fill" unless someone has a better suggestion